File: @LV1/1.3/dicecup/dicecup.go
package main

import "fmt"

func main() {
	var n, m int
	fmt.Scanf("%d %d", &n, &m)

	if m > n {
		m, n = n, m
	}

	for i := m + 1; i <= n+1; i++ {
		fmt.Println(i)
	}
}

File: LV1/1.3/dicecup/dicecup.cpp
#include <cstdio>

int main() {
    int n, m;
    scanf("%d %d", &n, &m);

    if (m > n) {
        int temp = m;
        m = n;
        n = temp;
    }

    for (int i = m + 1; i <= n + 1; i++) {
        printf("%d\n", i);
    }
}
